userspace/openpilot_python_dependencies.sh

```diff
@@ -3,12 +3,35 @@
 echo "installing uv..."
 
 export XDG_DATA_HOME="/usr/local"
-export CARGO_HOME="$XDG_DATA_HOME/.cargo"
+UV_BIN="$HOME/.local/bin"
```

Comment:
`UV_BIN` is not read by uv. With the new changes in 0.5.0 its now installed into `$XDG_DATA_HOME/../bin`, but `HOME` points to /root/ during the docker build. So `UV_BIN` points to a non existing folder.

Reply:
thanks for reviewing, i got a good understanding after looking over the [`0.5.0` Release notes](https://github.com/astral-sh/uv/releases/tag/0.5.0)

I also went ahead and removed `.cargo/bin` from PATH in the agnos Dockerfile since it serves no purpose now.